scripts/py_matter_yamltests/matter_yamltests/pseudo_clusters/clusters/log_commands.py

```diff
@@ -0,0 +1,26 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the 'License');
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an 'AS IS' BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from ..pseudo_cluster import PseudoCluster
+
+
+class LogCommands(PseudoCluster):
+    name = 'LogCommands'
+
+    async def UserPrompt(self, request):
+        pass
```

Comment:
I am okay with this as is for this PR. But in a follow up, or if you need to push an update anyways how do you feel about raising `NotImplementedError` for the time being? This should be guarded by a PICS code right?

Reply:
This things has been added into YAML but is really only uses by the Test Harness folks as a mechanism to split the test results into multiple parts and know if something has gone wrong. I would expect them to override it.

Reply:
Yeah, but if they forget to override it, they can get a false positive. If you raise and exception here it is harder to get that false positive